components/resource/scenemanager.cpp

```diff
@@ -710,6 +717,14 @@ namespace Resource
         mSharedStateMutex.unlock();
     }
 
+    void SceneManager::clearCache()
+    {
+        ResourceManager::clearCache();
+
+        OpenThreads::ScopedLock<OpenThreads::Mutex> lock(mSharedStateMutex);
+        mSharedStateManager->clearCache();
```

Comment:
This is AFAIK not currently used by the editor, but just in case we should clear the mInstanceCache also.